sdk/identity/identity/src/msal/browserFlows/msalBrowserCommon.ts

```diff
@@ -50,7 +50,10 @@ export function defaultBrowserMsalConfig(
     auth: {
       clientId: options.clientId!,
       authority,
-      knownAuthorities: getKnownAuthorities(tenantId, authority),
+      knownAuthorities: getKnownAuthorities(tenantId, authority, options.knownAuthorities),
+      authorityMetadata: options.authorityMetadata,
+      cloudDiscoveryMetadata: options.cloudDiscoveryMetadata,
+      skipAuthorityMetadataCache: options.skipAuthorityMetadataCache,
```

Comment:
Are these then used in MSAL directly?  I don't see any other use of them.

Reply:
Yes it is sent to MSAL directly